search/search-domain/src/main/java/io/camunda/search/entities/VariableEntity.java

```diff
@@ -15,7 +15,7 @@ public final record VariableEntity(
     String name,
     String value,
     String fullValue,
-    boolean isPreview,
+    Boolean isPreview,
```

Comment:
❓Why can it not be a primitive boolean?

Reply:
Because MyBatis always uses the Wrapper type in the resultMap. With the primitiv type, I get this error: 

`### SQL: SELECT       VAR_KEY,       PROCESS_INSTANCE_KEY,       SCOPE_KEY,       VAR_NAME,       VAR_VALUE,       TENANT_ID,       false as IS_PREVIEW     FROM VARIABLE           WHERE 1 = 1                        AND SCOPE_KEY IN        (  ? )                                               LIMIT ? OFFSET ?
### Cause: org.apache.ibatis.reflection.ReflectionException: Error instantiating class io.camunda.search.entities.VariableEntity with invalid types (Long,String,String,String,Boolean,Long,Long,String) or values (2,variable-name-48,variable-value-367,variable-value-367,false,1,3,tenant-937). Cause: java.lang.NoSuchMethodException: io.camunda.search.entities.VariableEntity.<init>(java.lang.Long,java.lang.String,java.lang.String,java.lang.String,java.lang.Boolean,java.lang.Long,java.lang.Long,java.lang.String)
`

Also when I use this as arg: 

`<arg column="IS_PREVIEW" javaType="boolean"/>`